src/Umbraco.Cms.Integrations.Automation.Zapier/Umbraco.Cms.Integrations.Automation.Zapier.csproj

```diff
@@ -15,6 +15,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
     <PackageReference Include="UmbracoCms.Web" Version="8.5.4" />
+	  <PackageReference Include="UmbracoCms.Core" Version="8.14.1" />
```

Comment:
I don't think it makes sense to have mixed versions here.  If we need 8.14,1 we should update the version of the existing `UmbracoCms.Web` reference,, bump the version of the package to a new minor, and make sure we set the dependency correctly when uploading the zip file to our.umbraco.com.